spec/unit/server_spec.rb

```diff
@@ -191,6 +191,55 @@ def app
         end
       end
     end
+
+    context "broken description" do
+      before do
+        store_raw_description("foo", <<-EOT
+        {
```

Comment:
Good morning @promulo, thank you for taking on this issue! 

Instead of defining a long description in the spec file itself, take a look at the ```system_description_factory.rb``` file. You can also create a broken example scope there to test this specific case.